hs_core/models.py

```diff
@@ -3383,12 +3430,18 @@ def can_view(self, request):
         """Pass through to abstract resource can_view function."""
         return AbstractResource.can_view(self, request)
 
-    def get_irods_storage(self):
+    def get_storage(self):
         """Return either IrodsStorage or FedStorage."""
-        if self.resource_federation_path:
-            return FedStorage()
-        else:
-            return IrodsStorage()
+
+        # if self.resource.storage_type == StorageCodes.IRODS:
+        return IrodsStorage()
```

Comment:
Note to come back here once I have looked over the implementation better.

Reply:
Linux storage is effectively turned off because this method always returns IrodsStorage.  Some of the commented-out code will have to un-commented to get this going.

Reply:
The Linux file storage option does not include a migration from irods to linux.  This is something that will have to be implemented if we decide to migrate existing resources from irods to linux.